docs/kb/error-codes-messages.md

```diff
@@ -1615,6 +1627,42 @@ requests.
 
 **Solution**: Resubmit the request using a valid [authentication](/#authentication) method.
 
+## [](#error-code-128)Error Code: 128
+
+```
+<rsp stat="fail" version="1.0">
+    <err code="128">Cannot create campaigns</err>
+</rsp>
+```
+
+**Problem**: Campaigns cannot be created for an account that has [Connected Campaigns](https://www.pardot.com/training/connected-campaigns-faq/) enabled.
+
+**Solution**: Create a campaign in [Salesforce](https://developer.salesforce.com/docs/api-explorer/sobject/Campaign).
+
+## [](#error-code-129)Error Code: 129
+
+```
+<rsp stat="fail" version="1.0">
+    <err code="129">Cannot modify campaign name or cost</err>
+</rsp>
+```
+
+**Problem**: Campaign name and cost cannot be modified via the API.
```

Comment:
recommend taking the contraction: Campaign name and cost can't be modified via the API.